.travis.yml

```diff
@@ -18,6 +18,7 @@ before_script:
   - git config --global user.name "Testy Testerson"
   - cp config.yaml.example config.yaml
   - gem install fpm --no-ri --no-rdoc
+  - export PATH="$(gem env gemdir)/bin:$PATH"
```

Comment:
`fpm` is the ruby program we use to build deb packages. We install it with rubygems, but the place `gem install` puts the executable isn't on our path. `gem env gemdir` tells us where the base install path is, and any executables will be in the `bin` subdir of that.

All of this is to fix the (unrelated) build failures in the first commit which are caused by travis not finding `fpm` when sideloader tries to use it.